python/ray/cross_language.py

```diff
@@ -22,9 +22,6 @@ def format_args(worker, args, kwargs):
     Returns:
         List of args and kwargs (if supported).
     """
-    if not worker.load_code_from_local:
```

Comment:
Would be better if you could delete all occurrences of `load_code_from_local` in the code base.

Reply:
Let's remove it in another pr